### ISSUE TITLE
is it possible to get TFserving tutorial for the existing pipeline and for the same data?

### ISSUE THREAD

**Issue by IamExperimenting** [Building-ML-Pipelines/building-machine-learning-pipelines#52]
# Thank you for reporting an issue!

If you want to report an issue with the code in this repository,
please provide the following information:

* Your operating system name and version, as well as version numbers of the following packages: tensorflow, tfx.
* Any details about your local setup that might be helpful in troubleshooting.
* Detailed steps to reproduce the bug.

If you found an error in the book, please report it at 
https://www.oreilly.com/catalog/errata.csp?isbn=0636920260912.

**Comment by MicAnt64** [comment]
I agree with the individual that opened this issue. The book becomes disjointed once one reaches the tf_serving chapter.
